base/src/com/google/idea/blaze/base/issueparser/BlazeIssueParser.java

```diff
@@ -49,10 +51,46 @@ public class BlazeIssueParser {
 
   private static final String BAZEL_BUILD_FILES_PATTERN = "(/.*?BUILD(?:\\.bazel)?)";
 
-  public static ImmutableList<BlazeIssueParser.Parser> defaultIssueParsers(
+  private static ImmutableList<Parser> commonParsers(Project project, WorkspaceRoot workspaceRoot, ProjectViewSet projectViewSet) {
+    return ImmutableList.<Parser>builder()
+            .add(
```

Comment:
Since Java 9, you can also use [`List.of(...)`](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/List.html#of(E...))